src/bin/pg_autoctl/cli_do_tmux.c

```diff
@@ -382,18 +464,21 @@ cli_do_tmux_script(int argc, char **argv)
 			tmux_add_send_keys_command(script,
 									   "%s do pgsetup wait --pgdata %s/node1",
```

Comment:
Good point. Thing is, we don't need to wait until Postgres is running, which could take quite a long time on a secondary because we then are waiting for `pg_basebackup` to complete first. So I have added a new command `pg_autoctl do tmux wait nodename`. When given a monitor, it waits until Postgres is running. When given a Postgres node, it waits until the state file exists on disk: we create the file once registered to the monitor. Seems to work well in my testing.